packages/react/src/components/GroupedList/GroupHeader.base.tsx

```diff
@@ -107,6 +109,9 @@ export class GroupHeaderBase extends React.Component<IGroupHeaderProps, IGroupHe
     if (!group) {
       return null;
     }
+
+    const groupLabel = `${group.name} (${group.count}${group.hasMoreData ? '+' : ''})`;
```

Comment:
not sure how well this will localize

Reply:
Good point! I copied it more or less from `_onRenderTitle` (minus the wrapping nodes), so I suppose it'll at least be consistent? 😅. Would it be better to just have the `group.count` after the name, and leave off the "+" sign?

Reply:
You should take in a function as props (passed to the `DetailsList` or whatever by the host) which accepts these fields and produces the final string:
```ts
const groupLabel = props.getGroupAriaLabel?({
  name: group.name,
  count: group.count,
  hasMoreData: group.hasMoreData
 });
If it's not present, I'm sorry but we should just use `ariaLabel` directly.

Reply:
@cschlechty / @ThomasMichon I updated the PR to use `aria-labelledby` pointing at the group title cell. It's still author-overridable through `group.ariaLabel`, and I think it sets a better default by avoiding separate visible vs. screen reader text.

Let me know if that addresses your concerns! :)